pages/common/git-lfs.md

```diff
@@ -26,3 +26,11 @@
 - Push all LFS objects to the remote server (useful if errors are encountered):
 
 `git lfs push --all {{remote_name}} {{branch_name}}`
+
+- Fetch all LFS objects:
+
+`git lfs fetch`
+
+- Checkout all Git LFS objects:
```

Comment:
Is there a technical difference between "Git LFS objects" and "LFS objects"? As far as I can tell, "objects" are already referring to "Git objects" in the repository.

Reply:
Git LFS Objects == LFS Objects, but Git Objects ≠ Git LFS Objects as far as I know.

Git LFS objects are stored separately to regular Git objects, with a reference to the LFS Object as a regular Git object - which gets replaced by `git-lfs` on checkout automatically (or, at least, it should - but aren't always, hence the edit).

Reply:
Okay, so can the examples be changed to use only one of the terms "Git LFS objects" and "LFS objects", in order to avoid ambiguity?